dnf5daemon-server/dbus/interfaces/org.rpm.dnf.v0.rpm.Rpm.xml

```diff
@@ -86,6 +86,24 @@ along with libdnf.  If not, see <https://www.gnu.org/licenses/>.
         <arg name="packages" type="aa{sv}" direction="out"/>
     </method>
 
+    <!--
+        list_fd:
+        @options: an array of key/value pairs
+        @file_descriptor: File descriptor opened for writing used to transfer data
+        @transfer_id: Identifier of this transfer. Used for signaling the end of transfer.
+
+        Get list of packages that match to given filters.
+        Packages are transfered in form of stream of JSON objects. Each JSON
+        object is a dictionary representing one package.
+
+        The same options as in list() method are supported.
```

Comment:
Do you want to deprecate the list option? At least it would be good to mention known issue about large data set.

Reply:
Not sure about deprecating - `list` provides much simpler interface then `list_fd` and should work for most use cases quite well. But I agree with describing possible issues with large data set.

I was also thinking about adding new option to the `list` (e.g. `limit`) to set a maximal number of returned packages. But this is out of scope of this PR.

Reply:
I think that data size is not about number of packages. Think that most of RPM string fields has no limitation of size.

Reply:
Is there a list of allowed package_attrs? May I ask for file lists?

Reply:
I added a commit with enhanced documentation covering these comments.